app/src/main/java/org/coepi/android/cen/Utils.kt

```diff
@@ -0,0 +1,103 @@
+package org.coepi.android.cen
```

Comment:
Generic utils classes tend to become bloated and should be avoided. These functions could be split in slightly more granular util classes, or even injectable dependencies.